index.js

```diff
@@ -513,7 +514,9 @@ export const Classes = {
 		selected: 'selected'
 	},
 	outcomes: {
+		availableAchievementScale: 'available-achievement-scale',
 		assessed: 'assessed',
+		clearAchievementThreshold: 'clear-achievement-threshold',
```

Comment:
I think this should be under `Actions` [here](https://github.com/Brightspace/d2l-hypermedia-constants/blob/864cb7f05bb43340008cfcfa1122a592435e4e64/index.js#L665-L669) instead shouldn't it